Dockerfile

```diff
@@ -38,27 +40,21 @@ ENV PIP_NO_CACHE_DIR true
 
 # Install python requirements
 COPY requirements requirements
-
-RUN yum -y install gcc && \
-    pip install -r requirements/local.txt -r requirements/deployment.txt && \
-    yum -y remove gcc && \
```

Comment:
I see we were previously installing and uninstalling gcc, why have we decided to leave gcc in the image now?

Reply:
Good question.  I moved the `gcc` install up with the main `yum install` in an attempt to speed up the build...and it does _a little_.  The install/uninstall was originally under a single `RUN` to try to slim the image size, but the image is already pretty massive, so it's not much of a win on that front.